apps/kserve/example/inference-service.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: serving.kserve.io/v1beta1
```

Comment:
The example folder should contain an example chart which is being deployed automatically in GitHub CI.

Reply:
Done.